tips/tests/tests.md

```diff
@@ -7,13 +7,32 @@
 # Wat moet ik gaan testen.
 
 ### Heeft geldige stadspas
-##### Als het product Minimafonds is en het besluit toekenning is dan is het True
-##### Als het product geen Minimafonds is en het besluit niet toekenning is dan is het False
+##### Als het product Minimafonds is en het besluit toekenning is, dan is het True.
+##### Als het product geen Minimafonds is en het besluit niet toekenning is, dan is het False.
 
 ### Is 18
-##### Als de geboortedatum 20-3-2002 is en de huidige datum is 7-5-2020 dan is het True
-##### Als de geboortedatum 20-3-2019 is en de huidige datum is 7-5-2020 dan is het False
+##### Als de geboortedatum 20-3-2002 is en de huidige datum is 7-5-2020, dan is het True.
+##### Als de geboortedatum 20-3-2019 is en de huidige datum is 7-5-2020, dan is het False.
 
+### Woont in gemeente Amsterdam
+##### Als mokum True is, dan hoort het True terug te geven.
+##### Als mokum False is, dan hoort het False terug te geven.
+
+### Heeft kinderen
+##### Als $.brp.kinderen True is, dan hoort het True terug te geven.
```

Comment:
Let erop dat kinderen een lege array (list) kan zijn en als er kinderen zijn, dan is het een array met objecten.